### PR TITLE
Adds Key Features

### LINKED ISSUE #422: [v3] Add key features
_This issue is part of the migration to v3. You can read full details about the various changes [here](https://github.com/exercism/v3-launch/)._

In Exercism v3, each track must specify exactly six "key features". Exercism uses these features to highlight the most interesting, unique or "best" features of a language to a student.

Key features are specified in the top-level `"key_features"` field in the track's `config.json` file and are defined as an array of objects, as specified in [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#key-features).

## Goal

The `"key_features"` field in the `config.json` file should be updated to describe the six "key features" of this track. See [the spec](https://github.com/exercism/v3-docs/blob/main/anatomy/tracks/config-json.md#key-features).

## Example

```json
{
  "key_features": [
    {
      "icon": "features-oop",
      "title": "Modern",
      "content": "C# is a modern, fast-evolving language."
    },
    {
      "icon": "features-strongly-typed",
      "title": "Cross-platform",
      "content": "C# runs on almost any platform and chipset."
    },
    {
      "icon": "features-functional",
      "title": "Multi-paradigm",
      "content": "C# is primarily an object-oriented language, but also has lots of functional features."
    },
    {
      "icon": "features-lazy",
      "title": "General purpose",
      "content": "C# can be used for a wide variety of workloads, like websites, console applications, and even games."
    },
    {
      "icon": "features-declarative",
      "title": "Tooling",
      "content": "C# has excellent tooling, with linting and advanced refactoring options built-in."
    },
    {
      "icon": "features-generic",
      "title": "Documentation",
      "content": "Documentation is excellent and exhaustive, making it easy to get started with C#."
    }
  ]
}
```

## Tracking

https://github.com/exercism/v3-launch/issues/5

### PR DESCRIPTION
**Pull request by rpottsoh**
resolves #422

What are the available "icon" options?  These don't appear to be documented.  For now I am essentially just borrowing what I found in csharp, similar enough.

### CODE REVIEW

**Comment by rpottsoh** [comment]
@ErikSchierboom, like #435, needs approval.  I am assuming that this will pass CI check once rebased after #435 is merged.

File: config.json
Comment (ErikSchierboom) [inline config.json:974]: This line slightly exceeds the maximum of 100 characters. Note that you _can_ merge this and fix later, as configlet does not yet have this check built in. Up to you :)

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**
